modules/server/src/main/java/org/jboss/wsf/stack/cxf/security/authentication/SubjectCreator.java

```diff
@@ -116,7 +127,7 @@ public Subject createSubject(SecurityDomainContext ctx, String name, String pass
             }
          }
 
-      } catch (RealmUnavailableException e) {
+      } catch (RealmUnavailableException | InvalidKeyException | InvalidKeySpecException | NoSuchAlgorithmException e) {
          throw MESSAGES.authenticationFailed(principal.getName());
```

Comment:
Is it better to log something if there is some exception thrown ?

Reply:
@jimma I've added some logging. Let me know if it is sufficient? Thanks!